bandpass/highpass.py

```diff
@@ -0,0 +1,171 @@
+import math
+import vertica_sdk
+
+"""
+An implementation of the High-pass filter as described by
+https://en.wikipedia.org/wiki/High-pass_filter 
+
+Setup:
+
+--- unfortunately, these filenames need to be an absolute pathname
+\set libfile '\'highpass.py\''
+--- also needs to be an absolute pathname
+\set data_csv '\'maw_tbrake_202104291444.csv\''
+
+drop library highpasslib cascade;
+CREATE LIBRARY highpasslib AS :libfile LANGUAGE 'Python';
+CREATE FUNCTION highpass AS LANGUAGE 'Python' NAME 'highpass_factory' LIBRARY highpasslib fenced;
+
+--- this only needs to be done once
+create flex table f_brake_data();
+copy f_brake_data from :data_csv parser fcsvparser();
+select compute_flextable_keys_and_build_view('f_brake_data');
+
+--- create table full of internal representation from view:
+create table bd as select tstamp, tbrakediscfl, tbrakediscfr, tbrakediscrl, tbrakediscrr from f_brake_data_view;
+
+--- let's start by creating a table with more convenient column names:
+create table b2 (row_n, tstamp, fl, fr, rl, rr) 
+as select row_number() over (order by tstamp),
+          tstamp, 
+          tbrakediscfl, 
+          tbrakediscfr, 
+          tbrakediscrl, 
+          tbrakediscrr
+from bd;
+
+--- now let's calculate deltas:
+--- https://www.vertica.com/blog/vertica-quick-tip-converting-intervals-numeric/ 
+--- teaches us a trick to turn an interval into a numeric by doing arithmetic on it:
+--- SELECT INTERVAL '24 HOURS' / INTERVAL '1 DAY' days;
+--- (returns 1)
+create table b3 (row_n, dt, dfl, dfr, drl, drr)
+as select row_n, 
+          (tstamp - lag(tstamp, 1) over (order by row_n)) / interval '1 second' seconds,
+          fl - lag(fl, 1) over (order by row_n),
+          fr - lag(fr, 1) over (order by row_n),
+          rl - lag(rl, 1) over (order by row_n), 
+          rr - lag(rr, 1) over (order by row_n) 
+from b2;
+
+--- Now, we've used SQL to calculate the folhighing:
```

Comment:
I'd like to see this comment before the command or just remove it because it's a little redundant.

folhighing???  Did you do a search and replace of low->high?